projects/plugins/protect/src/js/components/pricing-table/index.jsx

```diff
@@ -151,6 +162,11 @@ const ConnectedPricingTable = ( { onScanAdd } ) => {
 						isIncluded={ true }
 						label={ __( 'Check items against database', 'jetpack-protect' ) }
 					/>
+					<PricingTableItem
+						isIncluded={ true }
+						label={ __( 'Manual rules only', 'jetpack-protect' ) }
```

Comment:
Manual rules is appearing in the `Daily automated scans` and not in the `Web Application Firewall` line

Reply:
Fixed via [72ce74e](https://github.com/Automattic/jetpack/pull/29549/commits/72ce74e52612a3df862fbe4a9510d48290755e5d)